ctaplot/gammaboard/gammaboard.py

```diff
@@ -905,6 +911,44 @@ def update_gammaness_cut(change):
     return update_gammaness_cut
 
 
+def create_update_color(experiments_dict, fig_resolution, visible_experiments):
```

Comment:
small docstring would be nice if possible

Reply:
we probably need docstring for all the other functions too